templates/config_parameter_table.html

```diff
@@ -0,0 +1,179 @@
+<!-- Parameter/arguments table -->
```

Comment:
I think you MIGHT be able to remove all:

`<!-- ko if: Setting.findValue(Setting.BOTTOM_WINDOW_MODE) === Eagle.BottomWindowMode.GraphConfigAttributesTable -->`

within this file. I guess that this whole section of html will only be shown if the BOTTOM_WINDOW_MODE check on line 211 of base.html is 'true'.

So maybe BOTTOM_WINDOW_MODE === GraphConfigAttributesTable is guaranteed to be true within this file?

Reply:
good point, yep ill get rid of all of those